lib/public/views/Runs/Overview/exportRunsTriggerAndModal.js

```diff
@@ -18,25 +18,29 @@ import { runsActiveColumns } from '../ActiveColumns/runsActiveColumns.js';
  * Export form component, containing the fields to export, the export type and the export button
  *
  * @param {RunsOverviewModel} runsOverviewModel the runsOverviewModel
- * @param {array} runs the runs to export
+ * @param {Run[]} runs the runs to export
  * @param {ModalHandler} modalHandler The modal handler, used to dismiss modal after export
  *
- * @return {vnode[]} the form component
+ * @return {Component} the form component
  */
 const exportForm = (runsOverviewModel, runs, modalHandler) => {
     const exportTypes = ['JSON', 'CSV'];
     const selectedRunsFields = runsOverviewModel.getSelectedRunsFields() || [];
     const selectedExportType = runsOverviewModel.getSelectedExportType() || exportTypes[0];
     const runsFields = Object.keys(runsActiveColumns);
-    const enabled = selectedRunsFields.length > 0;
+    const loadingData = !runs;
+    const exportEnabled = selectedRunsFields.length > 0 && !loadingData;
+
+    // eslint-disable-next-line require-jsdoc
+    const optionalTruncatedInfo = () => runsOverviewModel.areExportableItemsTruncated
+        ? h(
+            '#truncated-export-warning.warning',
+            `The runs export is limited to ${runs.length} entries, only the last runs will be exported (sorted by run number)`,
+        )
+        : null;
```

Comment:
This message is not render immediately, I think during `loadingData`, spinner might be here instead